### ISSUE TITLE
Collection Date (Month) defaulting to 'January' in 'Loan Invoice Query1' query in Specify 7 (a11y) version

### ISSUE THREAD

**Issue by realVinayak** [specify/specify7#1823]
**Describe the bug**
The Collection Date (Month) menu is defaulting to 'January' in 'Loan Invoice Query1' query in the a11y version of Specify 7. The same query in edge version uses 'Any' as the month in Collection Date.

**To Reproduce**
1. log in to Specify 7 (a11y version) using spbirdadmin username and go to the Ornithology discipline.  
2. Go to the query builder and load the 'Loan Invoice Query1' query.
3. One of the queries is Loan Preparations -> Preparation -> Collection Object -> Collecting Information -> Collection Date (Month) -> Equal -> January
4. If you follow the same steps in Specify 7 edge version, using the same username and discipline and open the same 'Loan Invoice Query1', then the corresponding query is:  Loan Preparations -> Preparation -> Collection Object -> Collecting Information -> Collection Date (Month)(Any)

**Expected behavior**
In the a11y version, 'January' is selected by default from the dropdown menu. Instead, 'Equal' should be changed with 'Any' in the previous operator menu.

**Screenshots**
The left side of image is of edge version and the right side is of a11y version for the same query
![collection_date_month_bug_loan_invoice_query](https://user-images.githubusercontent.com/61122018/178763615-0c8faae8-3d2a-4ca3-a716-0f1c72413d47.PNG)

**Comment by maxxxxxdlp** [comment]
From @realVinayak:

> **Describe the bug** The Method menu is defaulting to 'By Hand' in 'Loan Invoice Query1' in Specify 7 (a11y) version. However, the Method for the same query in the edge version is set to 'Any'. The query link is: `specify/query/20`. The database is `Ornithology` and the username is `spbirdadmin`.
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. Log in to a11y version of Specify 7 using username `spbirdadmin`. Select `Ornithology `database and go to 'Loan Invoice Query1.
> 2. One of the queries is `Shipments -> Method -> Equal -> By Hand`.
> 3. Repeat the above steps in edge version, and the same query is `Shipments -> Method (Any)`
> 
> **Expected behavior** In the a11y version, the Method should be set to 'Any' instead of 'Equal to By Hand'.
> 
> **Screenshots** The left part is edge version and the right part is a11y version for the same query
> 
> ![shipment_method_loan_invoice_query](https://user-images.githubusercontent.com/61122018/178767256-c57b4496-ea8d-4e7c-b748-b9751c3e4e53.PNG)

**Comment by maxxxxxdlp** [comment]
From @realVinayak:

> **Describe the bug** The Title menu is defaulting to 'Asst. Director' in 'Loan Invoice Query1' query in Specify 7 (a11y) version. The same query in edge version has Title menu set to 'Any'. The query link is: `specify/query/20`. The username is `spbirdadmin` and the database is `Ornithology`.
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. Log in to a11y version of Specify 7 using username `spbirdadmin`. Select `Ornithology` database and go to 'Loan Invoice Query1' (`specify/query/20`).
> 2. One of the queries is Shipments -> Shipped To -> Title -> Equal -> Asst. Director
> 3. Repeat the above steps in the edge version and the same query is Shipments -> Shipped To -> Title (Any)
> 
> **Expected behavior** In the a11y version, the Title should be set to 'Any' instead of 'Asst. Director'
> 
> **Screenshots** The left part is edge version, and the right part is a11y version for the same query ![title_bug_loan_invoice_query](https://user-images.githubusercontent.com/61122018/178769916-f7d7ab84-70ad-41c2-8828-fb2d48cef8e9.PNG)

**Comment by maxxxxxdlp** [comment]
Thanks for finding this bug

A few general guidelines for new issues going forward:
- Thanks for specifying the query name and Specify version. Please also specify the name of the database the issue was found in. You can get the name of the database from the test server homepage
- Try to keep issue titles shorter. If you have further information, put it in the issue message. In this case, the following title would have been sufficient: `Collection Date (Month) defaulting to 'January' in Query Builder` or more generic: `Query Builder changes "any" picklists to their first value`

Thanks!
